sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -36,19 +36,26 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.4.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-netty</artifactId>
       <version>1.4.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <scope>test</scope>
```

Comment:
Netty dependency should not be test scoped.

Reply:
Done